bin/settings.ts

```diff
@@ -22,5 +22,5 @@ export const SETTINGS: HtsgetStatelessSettings & HtsgetStatefulSettings = {
     jwtAudience: [""], // Should match your cognito client id, i.e: 3jgmc7kqaaf8mqbv2sgmujslrp
   },
   // Enable additional features for compiling htsget-rs. `s3-storage` is always enabled.
-  features: ["experimental"], // i.e: Enables Crypt4Gh+htsget functionality
+  features: ["experimental", "s3-storage"], // i.e: Enables Crypt4Gh+htsget functionality
```

Comment:
Shouldn't this flag be `aws` instead of `s3-storage`?

Reply:
Yes! Well.. it depends on the version of htsget-rs. I was planning to pin it to a released version until new releases which introduce the "aws" flag. But it can be "aws" here anyway as it's just an example config.